tardis/io/model/readers/util.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import pandas as pd
+from radioactivedecay import Nuclide
+from radioactivedecay.utils import Z_DICT, elem_to_Z
+
+
+def read_csv_isotope_abundances(
```

Comment:
This is not *just* isotope abundances, correct?

Reply:
yes